mmv1/products/networkconnectivity/Group.yaml

```
@@ -0,0 +1,112 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Group
+legacy_long_form_project: true
+base_url: projects/{{project}}/locations/global/hubs/{{hub}}/groups
+create_url: projects/{{project}}/locations/global/hubs/{{hub}}/groups/{{name}}?updateMask=autoAccept.autoAcceptProjects
+self_link: projects/{{project}}/locations/global/hubs/{{hub}}/groups/{{name}}
+create_verb: 'PATCH'
+update_verb: 'PATCH'
+update_mask: true
+references:
+  guides:
+    'Official Documentation': 'https://cloud.google.com/network-connectivity/docs/network-connectivity-center/concepts/overview'
+  api: 'https://cloud.google.com/network-connectivity/docs/reference/networkconnectivity/rest/v1beta/projects.locations.global.hubs.groups'
+async:
+  type: 'OpAsync'
+  operation:
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result:
+    path: 'targetLink'
+  error:
+    path: 'error/errors'
+    message: 'message'
+examples:
+  - name: 'network_connectivity_group_basic'
+    primary_resource_id: 'primary'
+    vars:
+      hub_name: "hub1"
```

```suggestion
      hub_name: "network-connectivity-hub1"
```

With `-` inside the value of `hub_name`, a prefix `tf-test-` will be added. The prefix can be used to sweep any leaking resources.